README.md

```diff
@@ -111,6 +111,8 @@ Examples of how to utilize alpaka within CMake can be found in the `example` fol
 The whole alpaka library can be included with: `#include <alpaka/alpaka.hpp>`
 Code that is not intended to be utilized by the user is hidden in the `detail` namespace.
 
+Another library [*cupla*](https://github.com/ComputationalRadiationPhysics/cupla) provides a very CUDA-like user interface to alpaka.
```

Comment:
I would write this more personal, since we write cupla:
```suggestion
Furthermore, for a CUDA-like experience when adopting Alpaka we provide the library [*cupla*](https://github.com/ComputationalRadiationPhysics/cupla).
```

Reply:
I feel this is indeed better, changed and rebased. Thanks!